relay-server/src/services/store.rs

```diff
@@ -269,6 +284,26 @@ impl StoreService {
             }
         }
 
+        if let (Some(replay_event), Some(replay_recording)) = (replay_event, replay_recording) {
+            let combined_replay_kafka_message = Self::extract_combined_replay_kafka_message(
+                event_id.ok_or(StoreError::NoEventId)?,
+                replay_event,
+                replay_recording,
+                scoping,
+                start_time,
+                retention,
+            );
+            self.produce(
+                KafkaTopic::ReplayRecordings,
```

Comment:
I assume the recordings consumer will support the new format before we deploy relay?

Reply:
yes!